### PR TITLE
Eight new emotes for all to use

### PR DESCRIPTION
**Pull request by quackjack**
## About The Pull Request

Adds a few new emotes for avians and insects, as well as four different laughs from TF2.

## Why It's Good For The Game

Better emotes for races that don't really have much in that department, and TF2 funne laughs will be fun for the nerds.
(I noticed that we have a slight problem of having too many large sound files, so if you want to remove the sounds for the caws and chitters, or remove all the TF2 laughs, I understand.)

List:
- achitter
- mandible_twitch
- caw
- preen
- demolaugh
- engilaugh
- heavylaugh
- soldierlaugh

## Changelog
:cl:
add: Eight new emotes.
soundadd: Six new sounds to go alongside the emotes.
/:cl:

### CODE REVIEW

**Comment by MrFagetti** [comment]
Do you wanna make out? asking for a friend

File: modular_lumos/code/modules/mob/lumos_emotes.dm [hunk 1]
```diff
@@ -85,7 +85,129 @@
 
     user.nextsoundemote = world.time + 20
     playsound(user, pickweight(list(
-		'modular_lumos/sound/effects/fart.ogg' = 50, 
+		'modular_lumos/sound/effects/fart.ogg' = 50,
 		'modular_lumos/sound/effects/fart2.ogg' = 30,
 		'modular_lumos/sound/effects/fart3.ogg' = 5,
 		'modular_lumos/sound/effects/fart4.ogg' = 10)), 50, FALSE)
+    var/brap = rand(1,5)
+    user.atmos_spawn_air("miasma=[brap];TEMP=310.15")
```
Comment (coldud13) [inline modular_lumos/code/modules/mob/lumos_emotes.dm:93]: this was removed, why bring itback?

File: modular_lumos/code/modules/mob/lumos_emotes.dm [hunk 2]
```diff
@@ -85,7 +85,129 @@
...
+
+/datum/emote/living/achitter
+	key = "achitter"
+	key_third_person = "achitters"
```
Comment (coldud13) [inline modular_lumos/code/modules/mob/lumos_emotes.dm:97]: chitters

File: modular_lumos/code/modules/mob/lumos_emotes.dm [hunk 3]
```diff
@@ -85,7 +85,129 @@
...
+	if(user.nextsoundemote >= world.time)
+		return
+	user.nextsoundemote = world.time + 7
+	playsound(user, 'modular_lumos/sound/effects/caw.ogg', 50, 1, -1)
+
+/datum/emote/living/preen
+	key = "preen"
+	key_third_person = "preens"
+	message = "preens themselves."
+	message_param = "preens at %t."
```
Comment (coldud13) [inline modular_lumos/code/modules/mob/lumos_emotes.dm:133]: probably doesnt need a directed message

File: modular_lumos/code/modules/mob/lumos_emotes.dm [hunk 4]
```diff
@@ -85,7 +85,129 @@
...
+	emote_type = EMOTE_AUDIBLE
+	muzzle_ignore = FALSE
+	mob_type_allowed_typecache = list(/mob/living/brain, /mob/living/silicon, /mob/living/carbon/human)
+
+/datum/emote/living/manditwitch
+	key = "mandible_twitch"
+	key_third_person = "mandible_twitches"
+	message = "twitches their mandibles."
+	message_param = "preens at %t."
```
Comment (coldud13) [inline modular_lumos/code/modules/mob/lumos_emotes.dm:142]: copypasta

File: modular_lumos/code/modules/mob/lumos_emotes.dm [hunk 5]
```diff
@@ -85,7 +85,129 @@
...
+	muzzle_ignore = FALSE
+	mob_type_allowed_typecache = list(/mob/living/brain, /mob/living/silicon, /mob/living/carbon/human)
+
+/datum/emote/living/soldierlaugh/run_emote(mob/living/user, params)
+	if(!(. = ..()))
+		return
+	if(user.nextsoundemote >= world.time)
+		return
+	user.nextsoundemote = world.time + 7
+	playsound(user, 'modular_lumos/sound/effects/soldierlaugh.ogg', 50, 1, -1)
```
Comment (coldud13) [inline modular_lumos/code/modules/mob/lumos_emotes.dm:213]: yea nah

**Review by coldud13 — CHANGES_REQUESTED** [review]

**Comment by coldud13** [comment]
still need the tf2 emotes removed

File: modular_lumos/code/modules/mob/lumos_emotes.dm (hunk 1 above)
Comment (jjpark-kb) [reply modular_lumos/code/modules/mob/lumos_emotes.dm:93]: yea, this needs to stay out. I removed it because people are gamers.

**Comment by jjpark-kb** [comment]
also need to miasma removed

**Pull request closed without merging**